docker/annovar/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:bookworm-slim
```

Comment:
could we move this to `https://github.com/populationgenomics/images` to make use of the CI to build